common/thorhelper/thorcommon.cpp

```diff
@@ -1870,25 +1870,52 @@ void setAutoAffinity(unsigned curProcess, unsigned processPerMachine, const char
     if (numNumaNodes <= 1)
         return;
 
```

Comment:
I think its ok, it comes from start up script loop from 0 to processPerMachine-1
(processPerMachine == slavespernode)
But might be safe to check its range in the code.